metis-common/src/main/java/eu/europeana/metis/utils/ExternalRequestUtil.java

```diff
@@ -26,31 +32,46 @@
   private static final int MAX_RETRIES = 10;
   private static final int SLEEP_TIMEOUT = 500;
 
+  public static final Map<Class<?>, String> UNMODIFIABLE_MAP_WITH_NETWORK_EXCEPTIONS;
+
+  static {
```

Comment:
Why did you write this static block?

Reply:
So that I can initialize the final UNMODIFIABLE_MAP_WITH_NETWORK_EXCEPTIONS statically.
There are other alternatives for static initialization of course, I just chose to keep the same code that was used previously inside the WorkflowExecutor.
Does that make sense, or have you had something else in mind?

Reply:
Don't have anything else in mind. I don't believe I have seen a static block being used before so I was surprised to see it here. But now that I think of it, this makes sense. Thanks for answering!